app/models/backend_api.rb

```diff
@@ -33,6 +33,7 @@ class BackendApi < ApplicationRecord
   has_system_name(uniqueness_scope: [:account_id])
 
   scope :orphans, -> { joining { services.outer }.where { BabySqueel[:backend_api_configs].backend_api_id == nil } }
+  scope :of_product, ->(service) { joining { services }.where { BabySqueel[:backend_api_configs].service_id == service.id } }
```

Comment:
It would be nice to test this scope :)

Reply:
I will try the `service.backend_apis` :+1: